app/(auth)/signup.js

```diff
@@ -66,6 +70,15 @@ export default function SignUp() {
         uniqueDrills: [],
       });
 
+      try {
+        await sendEmailVerification(auth.currentUser);
```

Comment:
There seems to be an error here, when testing I kept having to hit the resend button to get the email to send and when I did, it would send 2 of them. I will test it again and update this comment if it works properly.

Reply:
Update, It still doesn't auto send, might be due to the redirect.

Reply:
Another update, after leaving it sitting while completing the PR review, the app crashed and I received this in my terminal 
![image](https://github.com/user-attachments/assets/63286064-606e-46fc-be47-511fc5f5f9fa)

Good news is the email sent 🤷‍♂️

Reply:
I'm not sure what would have caused these errors but let me know if it is working better as of the newest commits.